### PR TITLE
Fix the pre service action configuration test

### PR DESCRIPTION
**Pull request by bmhughes**
# Description

Move the configuration test into a ruby_block to ensure it is ran during convergence rather than during compilation.

## Issues Resolved

- None

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
         os:
           - 'centos-7'
           - 'centos-8'
-          - 'fedora-31'
+          - 'fedora-32'
```
Comment (bmhughes) [inline .github/workflows/ci.yml:45]: Before `fedora-latest` gets mentioned, the COPR repo doesn't have packages for Fedora 33 yet so it breaks testing.

Comment (Xorima) [inline .github/workflows/ci.yml:45]:
```suggestion
          - 'fedora'
```
Lets make this just match on fedora :+1:  then we won't have to worry in future?

Comment (bmhughes) [reply .github/workflows/ci.yml]: Must admit I'd not read the docs so didn't realise it could match on partials. Sounds a plan I'll stick that in.

**Review by Xorima — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [1.0.2](https://github.com/sous-chefs/syslog_ng/releases/tag/1.0.2)

**Pull request merged**